### PR TITLE
init:add package file

### PR DESCRIPTION
**Pull request by piyushagwl007**
Changes:
- Add package file and add actionhero as peerDependency
- Add gitignore file to ignore unnecessary files/folders to be added in git repo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "author": "Evan Tahler <evantahler@gmail.com>",
```
Comment (evantahler) [inline package.json]: You should be the author @piyushagwl007 !

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "name": "ah-swagger-plugin",
+  "version": "0.0.1",
+  "description": "A swagger plugin for the actionhero servers",
+  "main": "index.js",
```
Comment (evantahler) [inline package.json]: I don't think there's a "main" here, this package won't likely run on its own

File: package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "swagger",
+    "plugins"
+  ],
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "peerDependencies": {
+    "actionhero": ">=18.0.0"
+  },
+  "license": "Apache-2.0",
```
Comment (evantahler) [inline package.json:24]: If you are picking Apache 2, please also include a LENIENCE.TXT in the project

Comment (piyushagwl007) [reply package.json:24]: Alright

**Review by evantahler — APPROVED** [review]
👍

**Pull request merged**